Src/Test.UnitTests.Sarif.PatternMatcher.Sdk/FingerprintTests.cs

```diff
@@ -386,6 +386,23 @@ public void Fingerprint_ShouldParseNormally()
             fingerprint.ToString().Should().Be(fingerprintText);
         }
 
+        [Fact]
+        public void Fingerprint_ShouldMergeNormally()
+        {
+            string previousFingerprint = "[host=host][part=part]";
+            var fingerprint = new Fingerprint
+            {
+                Id = "id",
+                Secret = "secret"
+            };
+
+            fingerprint.Merge(previousFingerprint);
```

Comment:
Your test should also verify that no other values are unexpectedly non-null. :)

Reply:
thanks for the review!
with the new change, i'm checking all properties besides the one's that I already checked.